criu/arch/aarch64/crtools.c

```diff
@@ -135,3 +215,74 @@ int restore_gpregs(struct rt_sigframe *f, UserRegsEntry *r)
 
 	return 0;
 }
+
+int arch_ptrace_restore(int pid, struct pstree_item *item)
+{
+	PacAddressKeys *paca;
+	PacGenericKeys *pacg;
+	long pac_enabled_keys;
+	struct user_pac_address_keys upaca;
+	struct user_pac_generic_keys upacg;
+	struct iovec iov;
+	int ret;
+
+
+	pr_debug("%d: Restoring PAC keys\n", pid);
```

Comment:
What would happen if we create a checkpoint on a system with PAC keys and try to restore it on a system that does not support this? Would it make sense to check for `HWCAP_PACA` and show appropriate error message?